vmm/src/cpu.rs

```diff
@@ -342,12 +342,43 @@ pub struct CpuManager {
     vcpus_pause_signalled: Arc<AtomicBool>,
     reset_evt: EventFd,
     threads: Vec<thread::JoinHandle<()>>,
+    selected_cpu: u8,
 }
 
+const CPU_ENABLE_FLAG: usize = 1;
+
+const CPU_STATUS_OFFSET: u64 = 4;
+const CPU_SELECTION_OFFSET: u64 = 0;
+
 impl BusDevice for CpuManager {
-    fn read(&mut self, _base: u64, _offset: u64, data: &mut [u8]) {}
+    fn read(&mut self, _base: u64, offset: u64, data: &mut [u8]) {
+        match offset {
+            CPU_STATUS_OFFSET => {
+                // All CPUs are currently enabled
+                data[0] = 1 << CPU_ENABLE_FLAG;
```

Comment:
Definitely a nit at that point, but you probaly want a `|=` here.

Reply:
Fixed.